lib/WebService/SendBird/GroupChat.pm

```diff
@@ -0,0 +1,173 @@
+package WebService::SendBird::GroupChat;
+
+use strict;
+use warnings;
+
+use Carp;
+use WebService::SendBird::User;
+
+=head1 NAME
+
+WebService::SendBird::GroupChat - SendBird Group Chat
+
+=head1 SYNOPSIS
+
+ use WebService::SendBird::Group;
+
+ my $chat = WebService::SendBird::User->new(
+     api_client  => $api,
+     channel_url => 'chat_room_1',
+ );
+
+ $chat->update(name => 'Gossips');
+
+=head1 DESCRIPTION
+
+Class for SendBird Group Channel. Information about structure could be found at L<API Documentation|https://docs.sendbird.com/platform/group_channel>
+
+=cut
+
+use constant REQUIRED_FIELDS => qw(
+    api_client
+    channel_url
+);
+
+use constant OPTIONAL_FIELDS => qw(
+    is_broadcast
+    name
+    is_access_code_required
+    is_super
+    joined_member_count
+    is_public
+    cover_url
+    unread_mention_count
+    is_created
+    is_distinct
+    is_ephemeral
+    freeze
+    data
+    is_discoverable
+    last_message
+    sms_fallback_interval_sec
+    custom_type
+    unread_message_count
+    created_at
+    member_count
+    sms_fallback_enabled
+    max_length_message
+    members
```

Comment:
Should we only pick the fields we need? We are kind of making a deal here to keep this list updated whenever SB adds new fields on their side.

Reply:
I don't see a problem here. If list of fields will change in future, we can add new fields if we will need them, or somebody outside company could create PR to add new fields.